apps/vs-agent/src/utils/VsAgent.ts

```diff
@@ -111,7 +108,10 @@ export const createVsAgent = (options: VsAgentOptions): VsAgent => {
           }),
         ],
       }),
-      dids: new DidsModule({ resolvers: [new CachedWebDidResolver()] }),
+      dids: new DidsModule({
+        resolvers: [new CachedWebDidResolver()],
```

Comment:
Maybe not now, but at some point it will be needed to support resolving did:webvh as well. In that case, a WebVhDidResolver must be instanced as well.